rest/route/commit_queue.go

```diff
@@ -47,3 +47,47 @@ func (cq commitQueueGetHandler) Run(ctx context.Context) gimlet.Responder {
 
 	return gimlet.NewJSONResponse(commitQueue)
 }
+
+type commitQueueDeleteItemHandler struct {
+	project string
+	item    string
+
+	sc data.Connector
+}
+
+func makeDeleteCommitQueueItems(sc data.Connector) gimlet.RouteHandler {
+	return commitQueueDeleteItemHandler{
+		sc: sc,
+	}
+}
+
+func (cq commitQueueDeleteItemHandler) Factory() gimlet.RouteHandler {
+	return &commitQueueDeleteItemHandler{
+		sc: cq.sc,
+	}
+}
+
+func (cq commitQueueDeleteItemHandler) Parse(ctx context.Context, r *http.Request) error {
```

Comment:
Should this method return an error if ["project_id"] or ["item"] is not populated?

Reply:
the muxer won't ever deliver these requests so it can't ever be empty.